IPs/RSKIP271.md

```diff
@@ -27,8 +27,21 @@ After receiving a peg-out request, and if no previous request is waiting in the
 
 If [RSKIP272](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP272.md) is not activated, then the peg-out fees will be distributed evenly between all peg-out requests as it is today, and no change is required. 
 
-A new Bridge method `getEstimatedFeesForNextPegOutEvent()`is created so that users can estimate the peg-out fees. This method returns the fees of a peg-out transaction containing (N+2) outputs and 1.5 inputs, where N is the number of peg-outs requests waiting in the queue.  This method will be deprecated if RSKIP272 is activated.
+A new Bridge method `getEstimatedFeesForNextPegOutEvent()`is created so that users can estimate the peg-out fees. This method returns the fees of a peg-out transaction containing (N+2) outputs and 2 inputs, where N is the number of peg-outs requests waiting in the queue. This method will be deprecated if RSKIP272 is activated.
 
+### Peg-out Requests Count
+A new Bridge method `getQueuedPegoutsCount()` is created. This method returns the amount of peg-outs requests waiting in the queue to be released in the next peg-out transaction i.e. releaseRequestQueue size.
```

Comment:
```suggestion
A new Bridge method `getQueuedPegoutsCount()` is created. This method returns the amount of peg-outs requests waiting in the queue to be released in the next peg-out transaction.
```
I wouldn't mention anything related to the code

Reply:
Resolved